README.md

```diff
@@ -40,6 +40,42 @@ Task {
 }
 ```
 
+#### Offline initialization
+
+If you'd like to initialize Eppo's client without performing a network request, you can pass in a pre-fetched configuration JSON string.
+
+```swift
+try EppoClient.initialize(
+    configurationJson: "PRE-FETCHED-CONFIGURATION-JSON",
+    obfuscated: false
+);
+```
+
+The `obfuscated` parameter is used to inform the SDK if the flag configuration is obfuscated.
+
+The initialization method is synchronous and allows you to perform assignments immediately.
+
+**(Optional) Fetching the configuration from the remote source on-demand.**
+
+Combine both initialization methods to start your application with a local configuration and then 
+asynchronously fetch the latest flag configuration from the remote source.
+
+```swift
+try EppoClient.initialize(
+    configurationJson: "PRE-FETCHED-CONFIGURATION-JSON",
+    obfuscated: false,
+);
+
+...
+
+Task {
+    try await EppoClient.initialize(
+        sdkKey: "SDK-KEY-FROM-DASHBOARD",
+        forceReinitialize: true
+    );
+}
```

Comment:
This API is super-confusing

From the code alone, I would expect the second call to reset and loose initial configuration from the previous call. `forceReinitialize: true` also reinforces this expectation.

Reply:
UPD: found that it does indeed create a new client. This is problematic because:
- the new client does not have the configuration passed in JSON, so in this example, it would still be returning default assignments as if no json configuration were passed
- this invalidates the client returned from the first `initialize`. If user save that client, they must know to re-store the new reference. Otherwise, they would continue getting assignments from JSON.
- re-initialization creates a second assignment cache that is not shared with the first one